metecho/api/models.py

```diff
@@ -514,6 +516,15 @@ def finalize_status_completed(self, pr_number, *, originating_user_id):
         self.save()
         self.notify_changed(originating_user_id=originating_user_id)
 
+    def get_available_task_org_config_names(self, user):
+        from .jobs import available_task_org_config_names_job
+
+        available_task_org_config_names_job.delay(self, user=user)
```

Comment:
@wlonk I'd propose this should get called initially on project creation, and then we also provide an endpoint to refresh this list.